jest.config.ts

```diff
@@ -0,0 +1,88 @@
+import type {Config} from "jest";
+import os from "node:os";
+import pathPosix from "node:path/posix";
+
+
+let PRIMARY_REPORTER: string | [string, Record<string, unknown>] = "default";
+if ("undefined" !== typeof process.env.CI) {
+    PRIMARY_REPORTER = [
+        "github-actions",
+        {silent: false},
+    ];
+}
+console.log(`Environment variable "CI"="${process.env.CI}": ` +
+    `primary reporter will be ${JSON.stringify(PRIMARY_REPORTER)}`);
+
+const JEST_CONFIG: Config = {
+    collectCoverage: true,
+    collectCoverageFrom: ["src/**/*.{ts,tsx}"],
+    coverageProvider: "v8",
+    coverageReporters: ["text"],
+    coverageThreshold: {
+        "global": {
+            functions: 90,
+            lines: 90,
+        },
+        // eslint-disable-next-line no-warning-comments
+        // TODO: Remove / Adjust below overrides as more test cases are added.
+        "src/*.tsx": {
+            functions: 0,
+            lines: 0,
+        },
+        "src/components/": {
+            functions: 0,
+            lines: 0,
+        },
+        "src/contexts/": {
+            functions: 0,
+            lines: 0,
+        },
+        "src/services/": {
+            functions: 0,
+            lines: 0,
+        },
+        "src/typings/": {
+            functions: 0,
+            lines: 0,
+        },
+        "src/utils/": {
+            functions: 0,
+            lines: 0,
+        },
+        "src/utils/math.ts": {
+            functions: 100,
+            lines: 100,
+        },
+    },
+    displayName: {
+        name: "yscope-log-viewer",
+        color: "blue",
+    },
+    errorOnDeprecated: true,
+    maxConcurrency: os.cpus().length,
+    maxWorkers: "100%",
+    openHandlesTimeout: 1000,
+    reporters: [
+        PRIMARY_REPORTER,
+        "summary",
+    ],
+    showSeed: true,
+    testEnvironment: "node",
+    testMatch: [
+        pathPosix.join(__dirname, "test/**/?(*)test.{ts,tsx}"),
+    ],
+    testTimeout: 5000,
+    transform: {
+        ".(ts|tsx)$": [
+            "ts-jest",
+            {useESM: true},
```

Comment:
Where's this documented?

Reply:
This was from the generated file by `npx ts-jest config:init`. 
Ref: https://kulshekhar.github.io/ts-jest/docs/getting-started/installation

Reply:
Gotcha